### ISSUE TITLE
Sequent Relay Addition - 3 Relay Industrial

### ISSUE THREAD

**Issue by subcritical71** [rstrouse/relayEquipmentManager#47]
### What feature you'd like to see

Add support for the Sequent "Three 40A/240V Relays RS485 Daisy-channable HAT for Raspberry Pi" HAT [Link](https://sequentmicrosystems.com/collections/industrial-automation/products/three-40a-240v-relays-rs485-for-raspberry-pi).

## Motivation

I recently purchased the 3 relay HAT with the intention to use it for pool lighting control as well as chlorinator isolation. The relay is installed and partially functioning. Initially it seemed the unit would work using the 'Sequent 4rel4in v4' device. While it did work for simple switching there seems to be something going on when rapid switching is required, for example a lighting circuit, where it starts to trip over itself. The relay starts to cycle erratically when changing light shows. I will sometimes end with 3 different colors out of my 7 lights. And during the relay cycling they sound like they are galloping at irregular intervals.

## Pitch

This stackable 3 relay HAT could be used instead of 3 individual high load relays for various circuits reducing the room required in the cabinet and the complexity of the wiring. The relays can each handle 40A and 240V so just about any circuit could be controlled by this.

**Comment by rstrouse** [comment]
I'll try to get to this and the other sequent card this weekend.

**Comment by rstrouse** [comment]
I posted my first pass to the 3 relay card.  I don't have this card but try it out I think I read the command line code correct.
![image](https://github.com/rstrouse/relayEquipmentManager/assets/47839015/b1debbd6-9e93-42ac-90f8-ab22f288c966)

**Comment by subcritical71** [comment]
I will install tonight and test it in the morning and report back if any issues pop up. Thank you!

**Comment by subcritical71** [comment]
Tested it out this morning. Seems to be working good. I do get a spurious out of sync condition on the pool lights once in awhile. I have 6 lights and its usually the same 1 or 2 which wind up out of sync. Picking the same color clears it up. FYI, here is a screenshot of the REM screen which all relays de-energized. I cycled all the relays in various groupings and they all turned off/on as expected. Once again, thank you for doing this.

<img width="718" alt="Screen Shot 2023-05-20 at 11 15 59 AM" src="https://github.com/rstrouse/relayEquipmentManager/assets/12250429/b691102e-3576-4eab-81f5-ba423e9d3a9f">
